arm/Microsoft.Network/privateEndpoints/readme.md

```diff
@@ -32,17 +32,19 @@ The following resources are required to be able to deploy this resource:
 **Required parameters**
 | Parameter Name | Type | Description |
 | :-- | :-- | :-- |
-| `groupId` | array | Subtype(s) of the connection to be created. The allowed values depend on the type serviceResourceId refers to. |
-| `name` | string | Name of the private endpoint resource to create. |
+| `groupIds` | array | Subtype(s) of the connection to be created. The allowed values depend on the type serviceResourceId refers to. |
 | `serviceResourceId` | string | Resource ID of the resource that needs to be connected to the network. |
-| `targetSubnetResourceId` | string | Resource ID of the subnet where the endpoint needs to be created. |
+| `subnetId` | string | Resource ID of the subnet where the endpoint needs to be created. |
 
 **Optional parameters**
 | Parameter Name | Type | Default Value | Allowed Values | Description |
 | :-- | :-- | :-- | :-- | :-- |
+| `customDnsConfigs` | array | `[]` |  | Custom DNS configurations. |
 | `enableDefaultTelemetry` | bool | `True` |  | Enable telemetry via the Customer Usage Attribution ID (GUID). |
 | `location` | string | `[resourceGroup().location]` |  | Location for all Resources. |
 | `lock` | string | `'NotSpecified'` | `[CanNotDelete, NotSpecified, ReadOnly]` | Specify the type of lock. |
+| `manualPrivateLinkServiceConnections` | array | `[]` |  | Manual PrivateLink Service Connections. |
+| `name` | string | `[format('{0}-{1}', last(split(parameters('serviceResourceId'), '/')), parameters('groupIds')[0])]` |  | Name of the private endpoint resource to create. |
```

Comment:
The name default here doesn't align to the name default we are creating on resources.

Reply:
Updated the readme

Reply:
Will close when we close the comment on the PE name